src/index.js

```diff
@@ -22,6 +23,14 @@ export class AddonStore {
     this._preview = preview;
   }
 
+  getDatabase() {
+    return this._database;
+  }
+
+  setDatabase(database) {
```

Comment:
We need to consider about this method.
It's that, no addon should change this.
I mean abusing.